pt-BR/mail/chrome/messenger/migration/migration.dtd

```diff
@@ -16,19 +16,19 @@
 <!ENTITY importFromNothing.label        "Não importar nada">
 <!ENTITY importFromNothing.accesskey    "N">
 
-<!ENTITY welcome.title                  "Welcome to &brandShortName;">
-<!ENTITY tagline.label                  "A Better Way To Do Email">
-<!ENTITY thanks.label                   "Thanks for trying out &brandShortName;!">
-<!ENTITY quickStart1.label              "See the ">
-<!ENTITY quickStart2.label              "Quickstart Guide"> 
-<!ENTITY quickStart3.label              " to learn about all the features.">
-<!ENTITY start.label                    "Start using &brandShortName;">
-    
-<!ENTITY migration.title                "Account Migration Wizard">
-<!ENTITY migrationMac.title             "Account Migration Assistant">
-<!ENTITY importOptions.title            "Import Options">
+<!ENTITY welcome.title                  "Bem-vindo(a) ao &brandShortName;">
+<!ENTITY tagline.label                  "Um jeito melhor de enviar e-mails">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY tagline.label                  "Um jeito melhor de enviar correio eletrônico">